.pre-commit-config.yaml

```diff
@@ -16,7 +17,10 @@ repos:
     rev: 3.7.9
     hooks:
     -   id: flake8
-
-default_language_version:
-    # force all unspecified python hooks to run python2.7
-    python: python2.7
+        name: flake8 under python2
+        language_version: python2
+        exclude: ^doc/sphinx/sphinxext/
+    -   id: flake8
```

Comment:
It's not a problem to have 2 times the same id !?

Reply:
This is normal, we want to run the same checker under Python 2 and Python 3